src/ssl.c

```diff
@@ -43023,7 +43025,8 @@ int wolfSSL_i2d_X509_NAME(WOLFSSL_X509_NAME* name, unsigned char** out)
      *
      * returns the buffer size on success
      */
-    int wolfSSL_OBJ_obj2txt(char *buf, int bufLen, WOLFSSL_ASN1_OBJECT *a, int no_name)
+    int wolfSSL_OBJ_obj2txt(char *buf, int bufLen, const WOLFSSL_ASN1_OBJECT *a,
```

Comment:
Code below expects `a` to not be a const...

```
src/ssl.c:43109:17: error: passing 'char const[40]' to parameter of type 'void *' discards qualifiers [-Werror,-Wincompatible-pointer-types-discards-qualifiers]
                XMEMCPY(a->sName, buf, bufSz);
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Good catch.  I checked with Hide, this WOLFSSL_QT code block is no longer needed for Qt.  Has been removed.